pkg/actions/lua/lakefs/catalogexport/glue_exporter.lua

```diff
@@ -127,6 +129,14 @@ local function export_glue(glue, db, table_src_path, create_table_input, action_
         error("table " .. descriptor.type .. " in path " .. table_src_path .. " not supported")
     end
 
+    local create_db = opts.create_db or "no"
+    if create_db == "yes" or create_db == "if_not_exist" then
```

Comment:
This is quite confusing and error prone to type "enum" strings. 
The status should be: If  `opts.create_db_input` is not `nil` the user clearly want to call `glue.create_table` so there is no need in `yes/no` enums. 
Then, they can set `error_on_already_exists` to `false`  and that's it. 

So no need in `create_db`.

Reply:
OK so if I understood, you are suggesting the API
opts.create_db_input is nil by default, but if they specify something, then we create the DB
opts.error_on_db_exists to determine whether we swallow the error